Code sample - 23c with Spring Data JDBC

juarezjuniorgithub opened the pull request:
Code sample - 23c with Spring Data JDBC
Signed-off-by: Juarez Barbosa juarez.barbosa@oracle.com

juarezjuniorgithub commented:
@Kuassim @jeandelavarene I performed the adjustments you requested - indentation fixed. Please approve, thanks.

jeandelavarene commented:
Can you please use 23ai instead of 23c?

juarezjuniorgithub commented:
updated to 23ai as requested. @Kuassim please review and approve. thanks

jeandelavarene commented:
@Kuassim can you please follow up with @juarezjuniorgithub to ensure that all PR mention 23ai, not 23c and use the latest RU, i.e. 23.6. Thanks.

juarezjuniorgithub commented:
@Kuassim @jeandelavarene as requested, updated to 23ai + driver 23.6.0.24.10 as requested, updated to ucp17

Please approve, thanks. commit
https://github.com/oracle-samples/oracle-db-examples/pull/314/commits/ee339faba3c60531ada51bd9c66c9a31eb4cc4bd

jeandelavarene commented:
@juarezjuniorgithub please refresh your branch and update the PR.

juarezjuniorgithub commented:
@jeandelavarene, it was not my branch but an upstream one. As we talked on Slack, I did it for us. Please approve. Thanks again!
